src/lib/util/constants.ts

```diff
@@ -297,13 +297,17 @@ export const clientOptions: Partial<KlasaClientOptions> = {
 	i18n: {
 		defaultMissingKey: 'missingKey',
 		defaultNS: 'global',
-		i18next: {
-			preload: [...handlers.keys()],
+		i18next: (_: string[], languages: string[]) => ({
+			supportedLngs: languages,
+			preload: languages,
 			returnObjects: true,
 			returnEmptyString: false,
 			returnNull: false,
 			load: 'all',
+			lng: 'en-US',
```

Comment:
Is this necessary?

Reply:
Not necessary but at the same time, [the docs](https://www.i18next.com/overview/configuration-options) say it only affects automatic detection.